internals/daemon/api_test.go

```diff
@@ -92,7 +92,7 @@ func (s *apiSuite) TestSysInfo(c *check.C) {
 
 	sysInfoCmd.GET(sysInfoCmd, nil, nil).ServeHTTP(rec, nil)
 	c.Check(rec.Code, check.Equals, 200)
-	c.Check(rec.HeaderMap.Get("Content-Type"), check.Equals, "application/json")
+	c.Check(rec.Result().Header.Get("Content-Type"), check.Equals, "application/json")
```

Comment:
I happen to stumble on at least one location where ```rec.Header()``` is used after the ```ServeHTTP``` to check for header content.

```
rsp.ServeHTTP(rec, req)
	hdr := rec.Header()
	c.Check(hdr.Get("Content-Disposition") ...
```

Not sure if this is a big deal, or even not correct, but while reading the docs it seams for post response header validation the ```Result()``` method should be used, not ```Header()``` directly.

Reply:
Good catch -- fixed!